simulation.md

```diff
@@ -0,0 +1,456 @@
+
+
+## Changing particle masses / lifetimes/ widths
+Sometimes you need to change the mass or lifetime of a particle, either because the initial values are wrong, or the particle you actually want doesn't exist in EvtGen, and you need to adapt an existing particle.
+This can be done with python code inserted in the header:
+
+```
+# InsertPythonCode:
+#from Configurables import LHCb__ParticlePropertySvc
+#LHCb__ParticlePropertySvc().Particles = [ 
+# "N(1440)+              636       12212   1.0      1.4400000      2.194041e-24                 N(1440)+           21440      0.00",
+# "N(1440)~-             637      -12212  -1.0      1.4400000      2.194841e-24                   anti-N(1440)-           -21440      0.00",
+#"N(1520)+              420        2124   1.0      1.52000000      5.723584e-24                   N(1520)+           21520      0.00",
+# "N(1520)~-             421       -2124  -1.0      1.52000000     5.723584e-24                   anti-N(1520)-           -21520      0.00",
+#"N(1535)+              713       22212   1.0      1.53500000      4.388081e-24                   N(1535)+           21535      0.00",
+#"N(1535)~-             714      -22212  -1.0      1.53500000      4.388081e-24                   anti-N(1535)-           -21535      0.00",
+#"N(1720)+              775       32124   1.0      1.72000000      2.632849e-24                   N(1720)+           21720      0.00",
+#"N(1720)~-             776      -32124  -1.0      1.72000000      2.632849e-24                   anti-N(1720)-           -21720      0.00"
+#]
+# EndInsertPythonCode
+```
+The format is: 
+```
+#                    GEANTID    PDGID  CHARGE   MASS(GeV)      TLIFE(s)                    EVTGENNAME    PYTHIAID    MAXWIDTH
+```
+
+## Filtering a simulated sample
+
+For larger production requests, the amount of disk space required to store the sample becomes a problem. Therefore, a filtering of the final candidates obtained after the stripping step in the MC production can be applied. As this does not reduce the CPU requirements, filtering steps are best accompanied by a matching (but looser) set of generator cuts.
+
+Assuming we have a sample of simulated D*+ -> D0( -> K pi ) pi which we would like to filter on the Turbo line `'Hlt2CharmHadD02KPi_XSecTurbo'`:
+
+```python
+from GaudiConf import IOHelper
+IOHelper().inputFiles(
+   ['root://eoslhcb.cern.ch//eos/lhcb/grid/prod/lhcb/MC/2015/ALLSTREAMS.DST/00057933/0000/00057933_00000232_3.AllStreams.dst'],
+    clear=True)
+```
+
+We also do not need any events where the D0 candidate has a transverse momentum less than 3 GeV. We already know how to write the filter for this:
+```python
+from PhysSelPython.Wrappers import AutomaticData, SelectionSequence, Selection
+from Configurables import FilterDesktop
+
+line = 'Hlt2CharmHadD02KPi_XSecTurbo'
+Dzeros = AutomaticData('/Event/Turbo/'+line+'/Particles')
+
+pt_selection = FilterDesktop(
+    'D0_PT_selector', Code='PT')
```

Comment:
I'm not sure what a cut of just `PT` will do 😄 I guess you meant `PT > 3*GeV` or something.